installing/installing_azure/ipi/installing-azure-private.adoc

```diff
@@ -24,6 +24,13 @@ include::modules/installation-azure-user-defined-routing.adoc[leveloffset=+2]
 
 include::modules/installation-about-custom-azure-vnet.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin]
+
+* xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[Configuring your firewall]
+
```

Comment:
This is interesting, did these lines get added somehow?

Reply:
In both places, I added them to address the xrefs present in module `modules/installation-about-custom-azure-vnet.adoc`.

Reply:
Great, thank you